themes/default/layouts/page/pricing.html

```diff
@@ -200,801 +180,305 @@ <h2 class="text-3xl text-center mt-10 mb-0 lg:hidden">Pulumi Service</h2>
-            </div>
+            </li>
+        </ul>
 
-            <div class="compare-plans-table-row-container">
-                <div class="compare-plans-table-row-item">
-                    <span>Timeline</span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span><i class="fa fa-check pricing-checkmark mr-2"></i></span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span><i class="fa fa-check pricing-checkmark mr-2"></i></span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span><i class="fa fa-check pricing-checkmark mr-2"></i></span>
-                </div>
-            </div>
+        <p class="text-2xl font-bold lg:hidden mt-12 pl-5">Product</p>
+        <ul class="list-none p-0 mx-5 xl:ml-12 lg:hidden" data-faq-type="faq-product">
+            <li class="accordion-item text-2xl py-3">
+                {{ partial "accordian-header" (dict "text" "Can I use Pulumi for free?" "large_header" true) }}
 
-            <div class="compare-plans-table-row-container">
-                <div class="compare-plans-table-row-item">
-                    <span><a class="link" href="{{ relref . "/automation" }}">Automation API</a></span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span><i class="fa fa-check pricing-checkmark mr-2"></i></span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span><i class="fa fa-check pricing-checkmark mr-2"></i></span>
-                </div>
-                <div class="compare-plans-table-row-item">
-                    <span><i class="fa fa-check pricing-checkmark mr-2"></i></span>
+                <div class="accordion-item-body-no-animation text-base">
+                    <p>
+                        Yes! There are three ways to use Pulumi for free.
+                    </p>
+                    <p>
+                         First, the Pulumi Service is free to use, now and forever, for individuals.
+                         You get all of the convenience of automatic state management, unlimited
+                         deployments, and many other great features without needing to pay anything
+                         at all for it.
+                    </p>
+                    <p>
+                        Second, Pulumi is an <a class="underline" href="https://github.com/pulumi/pulumi">open source project</a>. You can
+                        <a class="underline" href="{{ relref . "/docs/intro/concepts/state#self-managed-backend" }}">run Pulumi entirely
+                        offline</a> without the online service’s features, and manage state yourself, instead of using the online service.
+                        There are no restrictions &mdash; it's all there in the open source for you to use freely as you'd like.
```

Comment:
"in the open source" feels like a mistake. Maybe this should just be "in the open"?

```suggestion
                        There are no restrictions &mdash; it's all there in the open for you to use freely as you'd like.
```